backend/app/controllers/reading_lists_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class ReadingListsController < ApplicationController
+  before_action :authorize_request
+
+  def create
+    if reading_list.valid?
+      render(json: { data: reading_list }, status: :created)
+    else
+      render(json: { errors: reading_list.errors.full_messages }, status: :unprocessable_entity)
+    end
+  end
+
+  def update
+    if reading_list.blank?
```

Comment:
```
reading_list = @user.reading_lists.create(reading_list_params)

if reading_list.valid?
```

Reply:
Ah! sorry should be `create`. 

For update it can be something like

```
if reading_list.update(reading_list_params)
```

Reply:
🤔 hmmm. lemme me if this is okay

```ruby
  def update
    if reading_list.blank?
      render(json: { errors: ["Reading list not found"] }, status: :not_found)
    elsif reading_list.update(reading_list_params)
      render(json: { data: reading_list }, status: :ok)
    else
      render(json: { errors: reading_list.errors.full_messages }, status: :unprocessable_entity)
    end
  end
```

I use the `.blank?` helper since using `find_by` in `@user.reading_lists.find_by(id: params[:id])` will return `nil` if not found